app/domain/errors.rb

```diff
@@ -367,6 +367,12 @@ module AuthnGcp
         msg:  "Role should have at least one of the following constraints: {0-constraints}",
         code: "CONJ00069E"
       )
+
+      InvalidResourceRestrictions = ::Util::TrackableErrorClass.new(
+        msg:  "Resource restriction '{0-resource-restriction-name}' does not match " \
+            "resource in GCP token",
+        code: "CONJ00072E"
+      )
```

Comment:
i suggest that we move error `CONJ00049E` to the JWT module and replace `Azure` with `JWT`. this way we can use it also here (and it will make sense)